index.js

```diff
@@ -2,7 +2,7 @@
 
 import chalk from 'chalk';
 import { exec } from 'child_process';
-import fs from 'fs';
+import fs from 'fs-extra';
```

Comment:
If you're going to use a package, it should be listed as a dependency in `package.json`. Although widely used, it's not safe to rely on dependencies' dependencies.

Reply:
I'd also like to know why this is necessary beyond the fs basics? It doesn't seem like we're doing anything fancy, and I like to keep dependencies to a minimum.

Reply:
It's because of the move and remove functions, there are some underlying issues especially with callbacks that fs has but fs-extra solves it.